airflow/models/dagrun.py

```diff
@@ -66,6 +66,13 @@ class DagRun(Base, LoggingMixin):
         UniqueConstraint('dag_id', 'run_id'),
     )
 
+    task_instances = relationship(
```

Comment:
Is it still used?

Reply:
It's not used...prolly will remove it.
@mik-laj  what is ur opinion, do we have it with a TODO sorta comment??